internal/ptx/snowflake.go

```diff
@@ -115,20 +116,7 @@ func (d *SnowflakeDialer) iceAddresses() []string {
 	if len(d.ICEAddresses) > 0 {
 		return d.ICEAddresses
 	}
-	return []string{
-		"stun:stun.voip.blackberry.com:3478",
-		"stun:stun.altar.com.pl:3478",
-		"stun:stun.antisip.com:3478",
-		"stun:stun.bluesip.net:3478",
-		"stun:stun.dus.net:3478",
-		"stun:stun.epygi.com:3478",
-		"stun:stun.sonetel.com:3478",
-		"stun:stun.sonetel.net:3478",
-		"stun:stun.stunprotocol.org:3478",
-		"stun:stun.uls.co.za:3478",
-		"stun:stun.voipgate.com:3478",
-		"stun:stun.voys.nl:3478",
-	}
+	return stuninput.AsSnowflakeInput()
```

Comment:
Rather than having two static copies of STUN targets (one for snowflake and one for stunreachability), I have created a single package that keeps those copies.